Create experts contact list (#17511)
nvaccess/nvda: projectDocs/community/expertsList.md

# Community experts list

This document outlines NV Access employees and community contributors.

> This can be used a reference to contact or tag developers in regards to relevant issues and pull requests.

Reviewer: ```suggestion
This can be used as a reference for people to contact or tag in regards to relevant issues and pull requests.
```

## NV Access

For optimal responses, please avoid tagging NV Access employees on unrelated tickets.

| GitHub | email | topics |
|---|---|---|
| @gerald-hartig | <gerald@nvaccess.org> | Controversial product decisions |
| @seanbudd | <sean@nvaccess.org> | Issue triage, technical decisions |
| @SaschaCowley | <sascha@nvaccess.org> | Issue triage, technical decisions |
| @michaelDCurran | <mick@nvaccess.org> | Advanced technical consulting |
| @Qchristensen | <quentin@nvaccess.org> | User documentation, user support |

> | @jcsteh | <jamie@nvaccess.org> | Firefox bugs |

Author: @jcsteh - confirming you're okay being added to this list, and can this be expanded to web browser bugs in general?

Reviewer 2: Thanks for the ping.

1. While I'm gratified to be considered an honorary NV Access employee, this isn't really correct these days. :) I just don't want to send the wrong message here, so I should probably be moved to community now.
2. Even though I'm involved in web standards and I do have Chromium contacts, I don't have any direct influence over what other browsers do and I am not a Chromium contributor, so I think we should leave this as Firefox. If a bug ends up being in Firefox and other browsers, I will likely be involved to some extent in figuring out what all browsers should do anyway, but if it's purely a Chromium bug, I probably can't help much.

## Community

| GitHub | email | topics |
|---|---|---|

> | | | |

Reviewer 3: ```suggestion
| @nvdaes | <nrm1977@gmail.com> | Add-on store, braille |
| | | |
```

Author: thanks @nvdaes !

# Community experts list

This document outlines NV Access employees and community contributors.
This can be used a reference to contact or tag developers in regards to relevant issues and pull requests.

## NV Access

> For optimal responses, please avoid tagging NV Access employees on unrelated tickets.

Author: ```suggestion
For optimal responses, please avoid tagging NV Access employees on unrelated tickets.
General enquiries should be directed to <info@nvaccess.org>.
```

| GitHub | email | topics |
|---|---|---|
| @gerald-hartig | <gerald@nvaccess.org> | Controversial product decisions |
| @seanbudd | <sean@nvaccess.org> | Issue triage, technical decisions |
| @SaschaCowley | <sascha@nvaccess.org> | Issue triage, technical decisions |

> | @michaelDCurran | <mick@nvaccess.org> | Advanced technical consulting |

Reviewer 4: ```suggestion
| @michaelDCurran | <mick@nvaccess.org> | Advanced technical consulting, Windows accessibility infrastructure and APIs |
```

Author: Appreciate the suggestion @codeofdusk - but @SaschaCowley and I are fine to tag on these topics

| @Qchristensen | <quentin@nvaccess.org> | User documentation, user support |
| @jcsteh | <jamie@nvaccess.org> | Firefox bugs |

## Community

| GitHub | email | topics |

> |---|---|---|

Reviewer 4: ```suggestion
|---|---|---|
| @codeofdusk | <codeofdusk@gmail.com> | Windows Terminal and console, Microsoft UI Automation |
```

Author: Thanks for adding your name 😄

# Community experts list

This document outlines NV Access employees and community contributors.

> This can be used as a reference for people to contact or tag in regards to relevant issues and pull requests.

Author: ```suggestion
This can be used as a reference for people to contact or tag in regard to relevant issues and pull requests.
```

## NV Access

For optimal responses, please avoid tagging NV Access employees on unrelated tickets.
General enquiries should be directed to <info@nvaccess.org>.

| GitHub | email | topics |
|---|---|---|
| @gerald-hartig | <gerald@nvaccess.org> | Controversial product decisions |
| @seanbudd | <sean@nvaccess.org> | Issue triage, technical decisions |
| @SaschaCowley | <sascha@nvaccess.org> | Issue triage, technical decisions |
| @michaelDCurran | <mick@nvaccess.org> | Advanced technical consulting |
| @Qchristensen | <quentin@nvaccess.org> | User documentation, user support |
| @jcsteh | <jamie@nvaccess.org> | Firefox bugs |

## Community

| GitHub | email | topics |
|---|---|---|
| @nvdaes | <nrm1977@gmail.com> | Add-on store, braille |

> | | | |

Author: ```suggestion
```

> | @nvdaes | <nrm1977@gmail.com> | Add-on store, braille |

Reviewer 5: I'm fine with being tagged on LibreOffice-related issues/PRs.

Just as an idea: Would it make sense to switch the order, i.e. list the topic first, then the contact person? (My expectation would be that people looking into this document would usually first search for the topic, then look into the other columns.)

```suggestion
| @nvdaes | <nrm1977@gmail.com> | Add-on store, braille |
| @michaelweghorn | <m.weghorn@posteo.de> | LibreOffice |
```

Author: Thanks @michaelweghorn 

Given how fragments and specific our expertise is, we want to generate this list first before grouping by topics
